src/redux/modules/mdb.js

```diff
@@ -490,7 +491,15 @@ const mdbSlice = createSlice({
   extraReducers: builder => {
     builder
       .addCase(ssrActions.prepare, onSSRPrepare)
-      .addCase(settingsActions.setContentLanguages, () => freshStore());
+      .addCase(settingsActions.setContentLanguages, () => freshStore())
+      .addMatcher(
```

Comment:
Not sure what this part is for...

Reply:
I think it would be right to read a bit about RTK :)